src/AtlasAdmin.ts

```diff
@@ -49,6 +54,25 @@ export class AtlasAdminManager {
     }
   }
 
+  async updateSynonyms(): Promise<void> {
+    console.log('Updating synonyms');
+
+    const synonymCollection = this.mongoClient.db(DB).collection<SynonymDocument>(SYNONYM_COLLECTION_NAME);
+
+    const synonymUpdates = getSynonymUpdateOperations('../synonyms.csv');
+
+    console.log('uploading the following parsed synonyms documents to Atlas: ', synonymUpdates);
+    try {
+      // we want to ensure that the primary property is a unique index
+      // so that we prevent duplicate synonym records.
+      // the createIndex method is idempotent, so calling it repeatedly will not cause side effects.
+      await synonymCollection.createIndex({ primary: 1 }, { unique: true });
+      await synonymCollection.bulkWrite(synonymUpdates);
```

Comment:
Does order of the updates matter? If not, I'd recommend making this an unordered bulk write since it can be slightly faster (in theory), especially if we have a lot of updates to perform